gatsby-node.js

```diff
@@ -115,13 +117,130 @@ exports.sourceNodes = async ({ actions }) => {
   return
 }
 
-// exports.onCreateNode = ({ actions, node }) => {
-//   const { type } = node.internal
-//   if (type !== `Member`) return
-//   const { bio } = node
-//   const bioAsHtml = markdownProcessor.process(bio)
-//   node.bio = bioAsHtml
-// }
+exports.onCreateNode = ({ node, actions }) => {
+  const { type } = node.internal
+  if (type === 'GhostAuthor') {
+    if (node.slug !== 'data-schema-author') {
+      actions.createNodeField({
+        node,
+        name: `member___NODE`,
+        value: `Labs__Member__${node.slug}`,
+      })
+    }
+  }
+}
+
+const createTagPages = (tags, createPage) => {
+  const tagsTemplate = path.resolve(`./src/templates/tag.tsx`)
```

Comment:
I think the other template is capitalized, so would recommend changing this to `...Tag.tsx`

Reply:
Actually you can see what I did at the other one, declared the template at the top of the file and then used it down here. Imo that way of doing it is a little cleaner / decouples operating on the file system